src/makeRobot.js

```diff
@@ -38,7 +38,80 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      let step = value;
+
+      if (step < 0) {
+        step = 0;
+      }
+
+      this.coords.y += step;
```

Comment:
My thinking this would be looks better
```suggestion
      if (value > 0) {
        this.coords.y += value;
      }
```